### ISSUE TITLE
Issues with numbers along with text

### ISSUE THREAD

**Issue by vinayakkailas** [TJCVRS/CRNN_Tensorflow#88]
Hi Is the model trained only on text images? because when i ran pretrained model on images with numbers and texts, numbers are getting recognized as random characters. If my training data has a lot of  cropped images with numbers in it and if i include the numbers in label also, after training will the model be able to recognize those numbers???

**Comment by TJCVRS** [comment]
@vinayakkailas Yes the model can be used for recognizing numbers
